main.rb

```diff
@@ -0,0 +1,52 @@
+def foo
+  a=b
+  return 1
+end
+def foo
+  a=b
```

Comment:
__[development][rubocop]__
* Method `Object#foo` is defined at both /sideci/sandbox2/main.rb:1 and /sideci/sandbox2/main.rb:6.
* Surrounding space missing for operator `=`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)